src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -107,7 +109,7 @@
     }
   }
 
-  .facility-result + .facility-result {
+  .facility-result+.facility-result {
```

Comment:
Is this a functional change or a linter fix? I'm not sure what rule our styleguide follows (if any) but I prefer the whitespace between the `+` operator.